targets/TARGET_Cypress/TARGET_PSOC6/cy_crc_api.c

```diff
@@ -0,0 +1,91 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "crc_api.h"
+#include "mbed_assert.h"
+#include "mbed_error.h"
+#include "cyhal_crc.h"
+#include "cyhal_crc_impl.h"
+
+#if DEVICE_CRC
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+static cyhal_crc_t cy_crc;
+static crc_algorithm_t cy_crc_cfg;
+static bool cy_crc_initialized = false;
+
+bool hal_crc_is_supported(const crc_mbed_config_t *config)
+{
+    return config->width <= 32;
+}
+
+void hal_crc_compute_partial_start(const crc_mbed_config_t *config)
+{
+    if (!cy_crc_initialized)
```

Comment:
can you fix coding style in HAL implementation like this? see https://os.mbed.com/docs/mbed-os/v5.12/contributing/style.html

Reply:
Applied astyle (`astyle -n --options=.astylerc targets/TARGET_Cypress/TARGET_PSOC6/*.*`): https://github.com/ARMmbed/mbed-os/pull/10692/commits/b9f7836ac2bbb48acf4336ba1be5aa0da41cb990

Added style conformance check to CY internal CI pipeline.